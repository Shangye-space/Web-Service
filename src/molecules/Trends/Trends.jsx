import React from 'react';
import classnames from 'classnames';

import BestSeller from '../../atoms/BestSeller';
import Recommendations from '../../atoms/Recommendations';

import styles from './Trends.module.scss';

const Trends = () => {
    return (
        <div className={classnames('container', styles.trendsWrapper)}>
            <div className={styles.header}>{"Today's Trends"}</div>
            <div className={classnames('col-12', styles.tileWrapper)}>
                <BestSeller />
                <Recommendations />
            </div>
        </div>
    );
};

export default Trends;
