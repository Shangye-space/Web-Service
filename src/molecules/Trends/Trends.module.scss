@import '../../settings/variables.scss';

.trendsWrapper {
    justify-content: center;
}

.header {
    text-align: left;
    font-family: $marker;
    font-size: 59px;
    line-height: 1.4em;
    padding: 0 30px;
    margin-bottom: 0.5em;
}

.tileWrapper {
    display: flex;
}
