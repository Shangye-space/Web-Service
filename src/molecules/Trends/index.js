import Trends from './Trends.jsx';

export default Trends;
