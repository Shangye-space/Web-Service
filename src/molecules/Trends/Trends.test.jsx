import React from 'react';
import ReactDOM from 'react-dom';
import Trends from './Trends.jsx';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Trends />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('snapshot renders', () => {
    const component = renderer.create(<Trends />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
