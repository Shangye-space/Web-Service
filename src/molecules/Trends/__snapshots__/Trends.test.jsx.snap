// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot renders 1`] = `
<div
  className="container trendsWrapper"
>
  <div
    className="header"
  >
    Today's Trends
  </div>
  <div
    className="col-12 tileWrapper"
  >
    <div
      className="col-6 bestSellerWrapper"
    >
      <img
        alt=""
        src="/images/989171832_o.jpg"
      />
    </div>
    <div
      className="col-6 wrapper"
    >
      <div
        className="banner"
      >
        <div>
          Our Recommendations
        </div>
      </div>
      <div
        className="items"
      >
        <div
          className="left"
        >
          <img
            src="/images/IMG_2592.jpg"
          />
        </div>
        <div
          className="right"
        >
          <img
            src="/images/RXTM-088-600x600.jpg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
